helm/templates/frontend/frontend-deployment.yaml

```
@@ -0,0 +1,34 @@
+  selector:
+    matchLabels:
+      app: frontend
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: frontend
+    spec:
+      containers:
+        - image:  {{ .Values.frontend.imageName }}:{{ .Values.frontend.imageVersion }}
```

YAMLlint (1.37.1)

[warning] too many spaces after colon

(colons)